### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by ishanivardhan** [NHSLeadership/nightingale-moodle#87]
@jdbrew , @Pro-Paul , @cehwitham - Pls contribute to this ticket to address any accessibility issues within the theme.
Feel free to remove yourselves from the assignee list once you have contributed from your end & assign/tag others as necessary. Thanks!

**Comment by penpadstudio** [comment]
Here are some standard accessibility criteria/perspectives suggested by W3C we should be trying to meet or at least be contemplating:

- [Video Captions](https://www.youtube.com/watch?v=iWO5N3n1DXU&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=1)
- [Good contrasting colours](https://www.youtube.com/watch?v=Hui87z2Vx8o&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=2)
- [Large links, Buttons & Controls](https://www.youtube.com/watch?v=CzfKB3PuuIY&index=3&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT)
- [Customisable text](https://www.youtube.com/watch?v=rbiI65Jcz5s&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=4)
- [Keyboard compatibility](https://www.youtube.com/watch?v=93UgG72os8M&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=5)
- [Clear Layout and Design](https://www.youtube.com/watch?v=tfkzj5VC9P8&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=6)
- [Notifications and Feedback](https://www.youtube.com/watch?v=E1fEv4Vpexg&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=7)
- [Text to Speech](https://www.youtube.com/watch?v=8Rn5pXCdZWU&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=8)
- [Understandable Content](https://www.youtube.com/watch?v=BYRxF2yInfA&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=9)
- [Voice Recognition](https://www.youtube.com/watch?v=7RHG_XiQ0ck&index=10&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT)
